src/cron/cleanZombieUsers.ts

```diff
@@ -8,10 +8,23 @@ const cron: Cron = {
   handler: async (con, logger) => {
     logger.info('cleaning zombie users...');
     const timeThreshold = subHours(new Date(), 1);
-    const { affected } = await con.getRepository(User).delete({
-      infoConfirmed: false,
-      createdAt: LessThan(timeThreshold),
-    });
+    const query = con
+      .createQueryBuilder()
+      .delete()
+      .from(User)
+      .where([
+        {
+          infoConfirmed: false,
+        },
+        {
+          emailConfirmed: false,
+        },
+      ])
+      .andWhere({
+        createdAt: LessThan(timeThreshold),
+      });
```

Comment:
```sql
DELETE FROM
  "public"."user"
WHERE
  (
    "infoConfirmed" = FALSE
    OR "emailConfirmed" = FALSE
  )
  AND "createdAt" < NOW() - INTERVAL '1 day'
```

Reply:
`infoConfirmed` will always be true for users signing up via email.

And `emailConfirmed` will always be true for users signing up via SSO.

So this will now catch zombies that never complete signup via email

Reply:
now that we have multi column match, might need index on this, if for any reason cron starts to be slow

Reply:
Currently no indexes on `infoConfirmed`, but can add for that and `emailConfirmed`.

Though currently the query takes ~750ms. When I add one index per column, it goes down to ~120ms

Reply:
yeah, don't think is needed right now, but just something to look for

Reply:
Added single index. Duration is the same as two indexes, but according to chatgippity, a single index is potentially better.